routes/markups.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express');
+const fs = require('fs')
+const fsPromises = require('fs').promises;
+const bodyParser = require('body-parser')
+
+let router = express.Router();
+
+router.get('/api/markups', async function (req, res, next) {
+    try {
+        const svgString = await fsPromises.readFile(`Markups.svg`, "utf8");
+        res.json({ status: 200, markups: svgString });
+    } catch (err) {
+        next(err);
+    }
+});
+
+router.post('/api/markups', bodyParser.json(), async function (req, res, next) {
```

Comment:
Express already has a [json body parser](https://www.geeksforgeeks.org/express-js-express-json-function/) built-in. Is there any reason for using the `body-parser` instead?